howso/details_stats.amlg

```diff
@@ -1,1057 +1,5 @@
-			filtered_values_map
-				(filter
-					(lambda (= (get (current_value) (list ".hyperparam_path" 0)) ".targetless") )
-					original_values_map
-				)
-		)
-		(if (= 1 (size filtered_values_map))
-			(conclude (first (values filtered_values_map)))
-
-			;revert because all were filtered out
-			(= 0 (size filtered_values_map))
-			(assign (assoc filtered_values_map original_values_map))
-
-			;else backup the filtered residuals map for further filtering below
-			(assign (assoc original_values_map filtered_values_map))
-		)
-
-		;leave only 'robust' residuals
-		(assign (assoc
-			filtered_values_map
-				(filter
-					(lambda (= (get (current_value) ".robust") (true)) )
-					filtered_values_map
-				)
-		))
-		(if (= 1 (size filtered_values_map))
-			(conclude (first (values filtered_values_map)))
-
-			;revert because all were filtered out
-			(= 0 (size filtered_values_map))
-			(assign (assoc filtered_values_map original_values_map))
-
-			;else backup the filtered residuals map for further filtering below
-			(assign (assoc original_values_map filtered_values_map))
-		)
-
-		;leave only 'robust' hyperparam residuals
-		(assign (assoc
-			filtered_values_map
-				(filter
-					(lambda (= (get (current_value) (list ".hyperparam_path" 2)) "robust") )
-					filtered_values_map
-				)
-		))
-		(if (= 1 (size filtered_values_map))
-			(conclude (first (values filtered_values_map)))
-
-			;revert because all were filtered out
-			(= 0 (size filtered_values_map))
-			(assign (assoc filtered_values_map original_values_map))
-
-			;else backup the filtered residuals map for further filtering below
-			(assign (assoc original_values_map filtered_values_map))
-		)
-
-		;leave only case weight of .none
-		(assign (assoc
-			filtered_values_map
-				(filter
-					(lambda (= (get (current_value) (list ".hyperparam_path" 3)) ".none") )
-					filtered_values_map
-				)
-		))
-		(if (= 1 (size filtered_values_map))
-			(conclude (first (values filtered_values_map)))
-
-			;revert because all were filtered out
-			(= 0 (size filtered_values_map))
-			(assign (assoc filtered_values_map original_values_map))
-
-			;else backup the filtered residuals map for further filtering below
-			(assign (assoc original_values_map filtered_values_map))
-		)
-
-		;leave only case weight of .case_weight
-		(assign (assoc
-			filtered_values_map
-				(filter
-					(lambda (= (get (current_value) (list ".hyperparam_path" 3)) ".case_weight") )
-					filtered_values_map
-				)
-		))
-		(if (= 1 (size filtered_values_map))
-			(conclude (first (values filtered_values_map)))
-
-			;revert because all were filtered out
-			(= 0 (size filtered_values_map))
-			(assign (assoc filtered_values_map original_values_map))
-
-			;else backup the filtered residuals map for further filtering below
-			(assign (assoc original_values_map filtered_values_map))
-		)
-
-		;at this point output the first one, whatever it happens to be
-		(first (values filtered_values_map))
-	)
-
```

Comment:
is the only method remaining in the file this !ProcessSelectedPredictionStats ? if so, a) is it still being used by anything and b) you should move it to wherever it's used get rid of this file altogether

Reply:
fair enough, I moved it.